src/Microsoft.AspNet.SignalR.Redis/RedisScaleoutOptions.cs

```diff
@@ -23,12 +23,12 @@ public RedisScaleoutOptions(string connectionString, string eventKey)
         {
             if (String.IsNullOrEmpty(connectionString))
             {
-                throw new ArgumentNullException("connectionString");
+                throw new ArgumentNullException(nameof(connectionString));
             }
 
             if (String.IsNullOrEmpty(eventKey))
             {
-                throw new ArgumentNullException("eventKey");
+                throw new ArgumentNullException(nameof(eventKey));
```

Comment:
From #15 
> This code should instead throw an `ArgumentException` with something like the fairly-common `Resources.ArgumentCannotBeNullOrEmpty`.

Should add the resource, fix the exception, and switch to `nameof` in one small commit.  @BrennanConroy